systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -248,6 +236,7 @@ function checkDirty(obj)
         num_q = obj.manip.num_positions;
         q=x(1:num_q); qd=x(num_q+(1:num_q));
         h = obj.timestep;
+        z_inactive_guess_tol = .01;
```

Comment:
Can we make this an settable  option? I imagine that it could have a performance impact depending on the speeds of the objects being simulated. Having dug through the Bullet source to modify tolerances like this, I'd hate to put our users through that.